benchmark/benchmark_multithreading_single.jl

```diff
@@ -7,7 +7,7 @@ using Benchmarks
 
 N = parse(Int, ARGS[1])
 grid = RectilinearGrid(size=(N, N, N), extent=(1, 1, 1))
-model = NonhydrostaticModel(architecture=CPU(), grid=grid)
+model = NonhydrostaticModel(architecture=CPU(); grid)
```

Comment:
architecture is not an argument anymore, it's inferred from the grid
```suggestion
model = NonhydrostaticModel(; grid)
```